kvm/src/lib.rs

```diff
@@ -1478,7 +1519,27 @@ mod tests {
 
     #[test]
     fn test_kvm_api_version() {
-        let kvm = Kvm::new().unwrap();
+        let kvm = Kvm::new(false).unwrap();
         assert_eq!(kvm.get_api_version(), KVM_API_VERSION as i32);
     }
+
+    impl PartialEq for CpuId {
+        fn eq(&self, other: &CpuId) -> bool {
+            let entries: &[kvm_cpuid_entry2] =
+                unsafe { self.kvm_cpuid[0].entries.as_slice(self.allocated_len) };
+            let other_entries: &[kvm_cpuid_entry2] =
+                unsafe { self.kvm_cpuid[0].entries.as_slice(other.allocated_len) };
+            self.allocated_len == other.allocated_len && entries == other_entries
```

Comment:
Does `==` explicitly compare member by member in the array?

Reply:
That's what I hope it does. But let me check and come back to you.

Reply:
Looks like `==` compares member by member.